axlearn/cloud/gcp/job.py

```diff
@@ -492,6 +497,24 @@ def __init__(self, cfg: Config):
             raise NotImplementedError(f"Missing system characteristics for {self._tpu_type}")
         super().__init__(cfg)
         self._output_volume_mount = dict(name="shared-output", mountPath="/output")
+        self._import_modules()
+        self.using_pathways = self._is_pathways_used()
+
+    def _is_pathways_used(self) -> bool:
+        # identify if a job is configured to use pathways by
+        # checking jax_backend flag and optional import for pathways utils
+        # brittle implementation
+        return (
+            "pathwaysutils" in self.config.import_modules
+            and "jax_backend proxy" in self.config.command
```

Comment:
Is "jax_backend proxy" in self.config.command sufficient? Why do we need `"pathwaysutils" in self.config.import_modules`?